tsdb/tx.go

```diff
@@ -316,7 +316,14 @@ func (l *LocalMapper) Begin(c *influxql.Call, startingTime int64, chunkSize int)
 	if fieldName != "" {
 		fid, err := l.decoder.FieldIDByName(fieldName)
 		if err != nil {
-			return fmt.Errorf("%s isn't a field on measurement %s", fieldName, l.job.MeasurementName)
+			switch {
+			case c.Name == "distinct":
```

Comment:
None of the rich messages are necessary, but they are nice, and already added.  Are you suggesting we take them out?